src/pages/Environments.js

```diff
@@ -195,7 +361,7 @@ const Environments = () => {
     await loadData()
   })
 
-  useOnMount(() => { loadData() })
+  useOnMount(loadData)
```

Comment:
I think this was probably intentional. Effect functions return a cleanup function which React will call when a dependency changes or the component is unmounted. An async effect function will return a Promise, which will cause an error when React attempts to call it as a function.

https://reactjs.org/docs/hooks-effect.html